### PR TITLE
MetricsHub - Hardware Dashboards

### PR DESCRIPTION
**Pull request by MarcDouguedroit**
Initial version of the MetricsHub hardware dashboards

### CODE REVIEW

**Comment by MarcDouguedroit** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MarcDouguedroit** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Comment by MarcDouguedroit** [comment]
@greatestusername-splunk Can you please review this PR?

**Comment by greatestusername-splunk** [comment]
Hi @MarcDouguedroit sorry for the wait. 

Would you mind adding a note to the bottom of your readme in the Support section you added to the effect:
`Splunk does not provide support for this dashboard and users should contact MetricsHub with any support requests`

We just need to make sure it's very clear that in this case this contribution is from outside of Splunk and we can't provide specific support for it.

Thanks!

**Comment by MarcDouguedroit** [comment]
> Hi @MarcDouguedroit sorry for the wait.
> 
> Would you mind adding a note to the bottom of your readme in the Support section you added to the effect: `Splunk does not provide support for this dashboard and users should contact MetricsHub with any support requests`
> 
> We just need to make sure it's very clear that in this case this contribution is from outside of Splunk and we can't provide specific support for it.
> 
> Thanks!

Hi @greatestusername-splunk 

I will update the readme with your suggested content.

We created detectors for MetricsHub. I'd like to include them in this PR.
Do you know if there is a document explaining the procedure to follow to export the detectors?

**Comment by greatestusername-splunk** [comment]
I think the best way to export detectors is still via the API as JSON. You can find the docs for that here:
https://dev.splunk.com/observability/reference/api/detectors/latest#endpoint-retrieve-detectors-query

File: detectors/metricshub/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Two options exist:
+1. Edit and send the Detector JSON [via API](https://dev.splunk.com/observability/reference/api/detectors/latest#endpoint-create-single-detector)
+  ```
+  curl -X POST "https://api.{REALM}.signalfx.com/v2/detector" \
+    -H "Content-Type: application/json" \
+    -H "X-SF-TOKEN: <value>" \
+    -d @"/path/to/detector/detector_name.json"
+  ```
+2. Copy the detector's JSON and paste it into your own Detector [via the UI](https://docs.splunk.com/Observability/alerts-detectors-notifications/create-detectors-for-alerts.html#nav-Create-detectors-to-trigger-alerts)
+
```
Comment (greatestusername-splunk) [inline detectors/metricshub/README.md:18]: Sorry Marc one last thing. Could you please add the disclaimer about support to this README as well? A link to your support or contact page may also be helpful in those disclaimers?

Thanks again for your contribution!

- [ ] Add Disclaimer for detectors

**Review by greatestusername-splunk — CHANGES_REQUESTED** [review]
Please add a disclaimer in the detector readme? Thanks!

File: detectors/metricshub/README.md (hunk 1 above)
Comment (MarcDouguedroit) [reply detectors/metricshub/README.md:18]: @greatestusername-splunk , I added a disclaimer, support information and a link to the MetricsHub web site.
I hope this is okay with you.

Thank you!

**Review by greatestusername-splunk — APPROVED** [review]

**Comment by greatestusername-splunk** [comment]
Thanks again for your contribution!

**Pull request merged**